src/codegate/logger/logger.py

```diff
@@ -0,0 +1,6 @@
+import structlog
+
+
+class OriginLogger:
+    def __init__(self, origin: str):
+        self.logger = structlog.get_logger().bind(origin=origin)
```

Comment:
I think this needs to be
```
self.logger = structlog.get_logger("codegate").bind(origin=origin)
```

Or at least with this change I'm getting `debug` logs even though I set in the cli the logging level to INFO
```
2024-12-18 12:33:09 [debug    ] Processing body through pipeline: 0 bytes origin=copilot_proxy
2024-12-18 12:33:09 [debug    ] No pipeline strategy selected  origin=copilot_proxy
2024-12-18 12:33:09 [debug    ] Extracting path from /extensions/chat.json origin=copilot_proxy
2024-12-18 12:33:09 [debug    ] Processing body through pipeline: 0 bytes origin=copilot_proxy
2024-12-18 12:33:09 [debug    ] No pipeline strategy selected  origin=copilot_proxy
```